README.md

```diff
@@ -1,2 +1,20 @@
-# KOBConnect
-Wallet Application for a KOB VON Network Member
+Title     | KOBConnect   
+----------| -------------
+Version   | _TBD_        
+## Abstract
+KOBConnect is the digital wallet application. The users can use it to store credentials and can access these stored credentials with ease. The wallet application is going to come with basic set of authentication to provide security for the user. The user can store anything useful.
+## Dependent Projects
+KOBWallet is a web application of KOBConnect app

```

Comment:
This point is incorrect and the name of the association wallet is KOB Association Wallet(KOBAW). The association wallet is used by associations and the KOBConnect app is  used by the citizens of Kochi.